### ISSUE TITLE
Some stuff are deprecated.

### ISSUE THREAD

**Issue by abduelhamit** [jamischarles/atom-todo-show#34]
Use the `atom-workspace` tag instead of the `workspace` class.
```
C:\Users\Abdülhamit\.atom\packages\todo-show\keymaps\show-todo.cson:10:2
```
---
Style elements within text editors using the `atom-text-editor::shadow` selector or the `.atom-text-editor.less` file extension. If you want to target overlay elements, target them directly or as descendants of `atom-overlay` elements.
```
C:\Users\Abdülhamit\.atom\packages\todo-show\stylesheets\show-todo.less:?:?
```

**Comment by framerate** [comment]
@jamischarles Do you want me to issue a pull request for this or are you on it? It should only take a few minutes (looks like just a find/replace)

**Comment by jamischarles** [comment]
Go for it! Would be happy to merge.

**Comment by framerate** [comment]
@jamischarles Does this one make sense to you?

```
Style elements within text editors using the `atom-text-editor::shadow` selector or the `.atom-text-editor.less` file extension. If you want to target overlay elements, target them directly or as descendants of `atom-overlay` elements.
```

**Comment by abduelhamit** [comment]
@framerate Look at joefitzgerald/go-plus#132. They had, talked about and fixed the same warning message.

**Comment by framerate** [comment]
Issued a PR: https://github.com/jamischarles/atom-todo-show/pull/38

Note: I removed some event handlers here:

https://github.com/framerate/atom-todo-show/commit/a3110531a9eeceb8a930f3763c1b209e69f4e2c2#diff-c1bec0e2a183505f86460299897eb228L236

I wasn't sure what they were doing! Please take a look and let me know if they're doing something I may not be using? (they were throwing deprecation errors due to spacepen not having the same exposed handlers as the old atom view)

**Comment by jamischarles** [comment]
That looks like it was left over boilerplate. Doesn't appear to affect anything.

**Comment by jamischarles** [comment]
Fixed with #38
